[Cases] Add new sub feature privilege to prevent access to case settings (#4557)
elastic/security-docs: docs/getting-started/cases-req.asciidoc

[[case-permissions]]
= Cases prerequisites

:frontmatter-description: Learn about the {kib} feature privileges required to access {elastic-sec} cases. 
:frontmatter-tags-products: [security]
:frontmatter-tags-content-type: [how-to] 
:frontmatter-tags-user-goals: [configure]

> 

Reviewer: I wasn't aware that we're adding frontmatter to ESS doc files. Is this something we should be doing for all files?

Author: Probably a good idea to double-check with @colleenmcginnis to be sure, but it was originally something that helped fill in the frontmatter for migration.

* **All** for the *Cases* feature under *Security*
* **All** for the *{connectors-feature}* feature under *Management*

[NOTE]
====

> Roles without **All** *{connectors-feature}* feature privileges cannot create, add, delete, or modify case connectors.

Reviewer 2: No issue with the sentence structure. This suggestion is mainly to improve clarity. The way the text is styled, it sort of looks like we're telling users that they need feature privs for the All Actions and Connectors feature (see example below). 

<img width="400" alt="Screenshot 2024-01-08 at 3 29 02 PM" src="https://github.com/elastic/security-docs/assets/79124755/78fa419d-594e-4bc5-a65e-e9fd2a6a90e1">

```suggestion
Roles without **All** privileges for the *{connectors-feature}* feature cannot create, add, delete, or modify case connectors.
```

> By default, `All` for the *Cases* feature includes authority to delete cases and comments and edit case settings unless you customize the sub-feature privileges.

Author: ```suggestion
By default, `All` for the *Cases* feature includes authority to delete cases, delete alerts and comments from cases, and edit case settings unless you customize the sub-feature privileges.
```

> By default, `All` for the *Cases* feature includes authority to delete cases, delete alerts and comments from cases, and edit case settings unless you customize the sub-feature privileges.

Reviewer 2: It might make sense to switch this with the previous sentence about roles without actions and connector privs. That way the note lines up with the order of the bullets. Also made a minor edit to break up the sentence a bit.

```suggestion
By default, **All** for the *Cases* feature allows you to delete cases, delete alerts and comments from cases, and edit case settings. Customize the sub-feature privileges to limit feature access.  
```

Reviewer: Agree with Nastasha's comment, just a tiny tweak to the suggestion, to make it clearer that this action is optional.
```suggestion
By default, **All** for the *Cases* feature allows you to delete cases, delete alerts and comments from cases, and edit case settings. You can customize the sub-feature privileges to limit feature access.
```

NOTE: Before a user can be assigned to a case, they must log into {kib} at least
once, which creates a user profile.

| Give view-only access for cases
a| **Read** for the *Security* feature and **All** for the *Cases* feature

> NOTE: By default, `Read` for the *Cases* feature does not include authority to delete cases or delete alerts and comments from cases. You also cannot view or edit case settings. You can eanble these actions by customizing the sub-feature privileges.

Author: ```suggestion
NOTE: By default, `Read` for the *Cases* feature does not include authority to delete cases or delete alerts and comments from cases. You also cannot view or edit case settings. You can enable these actions by customizing the sub-feature privileges.
```

> NOTE: By default, `Read` for the *Cases* feature does not include authority to delete cases or delete alerts and comments from cases. You also cannot view or edit case settings. You can enable these actions by customizing the sub-feature privileges.

Reviewer 2: I feel as though first sentence might a bit redundant since the note is tied to a single bullet. WDYT about removing it and making the phrasing more positive (i.e., "you can do this..." instead of "by default, you can't do this...")?

```suggestion
NOTE: Customize the sub-feature privileges to allow access to deleting cases, deleting alerts and comments from cases, and viewing or editing case settings. 
```

Reviewer: Same as above, tiny tweak to Nastasha's suggestion.
```suggestion
NOTE: You can customize the sub-feature privileges to allow access to deleting cases, deleting alerts and comments from cases, and viewing or editing case settings.
```
